apiVersion: openapi/v1alpha1
kind: Radar
metadata:
    name: "16490727"
    labels:
        datastore_id: "5847000167743488"
data:
    classification: Serious Bug
    created: "2014-04-01T21:34:42.363443Z"
    description: "Not sure how this happens, but when clicking for a second time on my \"Restore\" button, here is what comes in the console, while neither the success nor failure method is called (thus \"infinite\" download).\r\n\r\n === __61-[CKClientDispatch _sendMessage:callbackQueue:callbackBlock:]_block_invoke_2: Got error (Connection invalid)."
    email: mc@stuffmc.com
    modified: "2014-04-01T21:34:42.363649Z"
    number: "16490727"
    number_intvalue: 16490727
    originated: "2014-04-01"
    parent_number: '&{NULL_VALUE}'
    product: OS X SDK
    product_version: 10.9.2
    reproducible: Always
    resolved: ""
    status: Open
    title: CKClientDispatch Got error (Connection invalid).
